docs/pages/debugging/tools.mdx

```diff
@@ -10,9 +10,75 @@ import { Terminal } from '~/ui/components/Snippet';
 
 React Native consists of both JavaScript and native code. Making this distinction is very important when it comes to debugging. If an error is thrown from the JavaScript code, you might not find it using debugging tools for native code. This page lists a few tools to help you debug your Expo project.
 
+## Developer menu
+
+The **Developer menu** provides access to useful debugging functions. It is built into dev clients and Expo Go. If you are using an emulator, simulator, or have a device connected via USB, you can open this menu by pressing <kbd>m</kbd> in the terminal where Expo CLI has started the development server.
+
+<Collapsible summary="Alternative options to open the Developer menu">
+
+- Android device (without USB): Shake the device vertically.
+- Android Emulator or device (with USB):
+
+  - Press <kbd>Cmd ⌘</kbd> + <kbd>m</kbd> or <kbd>Ctrl</kbd> + <kbd>m</kbd>
+  - Run `adb shell input keyevent 82`
+
+- iOS device (without USB):
+  - Shake the device.
+  - Touch 3 fingers to the screen.
```

Comment:
```suggestion
  - Touch three fingers to the screen.
```

Numerals are written as words usually in context of gestures:
* https://support.apple.com/en-us/102482